src/style.css

```diff
@@ -0,0 +1,138 @@
+body {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.item {
+  width: 200px;
+  padding: 30px 14px 15px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+}
+
+.item__image {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 45px;
+}
+
+.item__name {
+  display: inline-block;
+  height: 31px;
+  margin: 0 0 5px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  color: #000;
+}
+
+.item__code {
+  height: 13px;
+  margin-bottom: 14px;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+  color: #000;
+}
+
+.item__rating {
+  height: 15px;
+  margin-bottom: 19px;
+  display: flex;
+  justify-content: space-between ;
+  align-items: flex-end;
+}
+
+.item__stars {
+  display: flex;
+}
+
+.star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background: url(./images/star.svg) no-repeat center;
+}
+
+.item__stars :not(:last-child) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
```

Comment:
use your code from previous task, I should have possibility to change class in HTML, and count of active stars should also change